### PR TITLE
CF-600 Make swift generation optional in generate_load and tests.

### PR DESCRIPTION
**Pull request by plomakin**
Added SWIFT_TESTING_ENABLED variable to config.py

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by SVilgelm** [comment]
👍

File: cloudferry_devlab/cloudferry_devlab/tests/config.py [hunk 1]
```diff
@@ -28,6 +28,9 @@
                       '192.168.1.8': 'juno'}
 SWIFT_AUTH_VERSION = '2'
 
+SKIP_SWIFT_GENERATION = True
```
Comment (roman-verchikov) [inline cloudferry_devlab/cloudferry_devlab/tests/config.py]: Let's rename to `SWIFT_TESTING_ENABLED` and invert the `if` statement above

**Comment by SVilgelm** [comment]
retest pep8

**Comment by MirantisWorkloadMobility** [comment]
Pylint failed. See message above.
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-pylint/2857/
```
************* Module cloudferry_devlab.generate_load
cloudferry_devlab/cloudferry_devlab/generate_load.py:1175: [E1101(no-member), Prerequisites.run_preparation_scenario] Module 'cloudferry_devlab.tests.config' has no 'SKIP_SWIFT_TESTING' member
```

**Comment by roman-verchikov** [comment]
+1

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by SVilgelm** [comment]
retest pep8

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
Unittests failed.
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests/2/

**Comment by MirantisWorkloadMobility** [comment]
Unittests success.

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**